data/comparator_dictionary.yaml

```
@@ -0,0 +1,139 @@
+version:
+  - 1.0
```

Does not need to be a list:

```suggestion
  - 1.0
```